docs/releases.rst

```diff
@@ -3,6 +3,78 @@
 Release notes
 =============
 
+Version 2.1.4
+-------------
+
+**New features**
+
+- Improve ``pg_rewind`` behavior on typical Debian/Ubuntu systems (Gunnar "Nick" Bluth)
+
+  On Debian/Ubuntu systems it is common to keep Postgres config files outside of the data directory. That makes ``pg_rewind --restore-target-wal`` to fail because it can't find ``postgresql.conf`` when trying to figure out``resolve_command``.
+
+- Allow setting ``TLSServerName`` on Consul service checks (Michael Gmelin)
+
+  Useful in case checks are done by IP and the Consul ``node_name`` is not an FQDN.
+
+- Added ``ppc64le`` support in watchdog (Jean-Michel Scheiwiler)
+
+  And fixed watchdog support on some non-x86 platforms.
+
+- Switched aws.py callback from ``boto`` to ``boto3`` (Alexander Kukushkin)
+
+  The ``boto`` is abandoned since 2018 and fails in python 3.9.
+
+- Periodically refresh service account token on K8s (Haitao Li)
+
+  Since Kubernetes v1.21 service account tokens expire in 1 hour.
+
+- Added ``/read-only-sync`` monitoring endpoint (Dennis4b)
+
+  It is similar to the ``/read-only`` but includes only synchronous replicas.
+
+
+**Stability improvements**
+
+- Don't copy the logical slot if there is a mismatch with the config (Alexander)
+
+  Previously Patroni was copying a logical slot to a replica with restart, noticing that ``plugin`` or ``database`` are different from the config and immediately removing it.
```

Comment:
If someone does `patronictl edit-config` and changes for example the `database` for an existing logical slot the following happens:
- primary will notice that the existing slot is created in the "wrong" database and try to remove it. If the slot is active `pg_drop_replication_slot()` will fail.
- replica will notice that the existing slot is created in the "wrong" database and removes it. After that it connects to the primary, gets the replication slot file, stops postgres, writes the slot file to PGDATA, and starts postgres. But, since the slot on the primary exists in the "wrong" database, slot on the replica is also wrong, and the action repeates.

Reply:
Thanks @CyberDem0n , here is my attempt to rephrase it:
```suggestion
  A replica won't copy a logical replication slot from the primary anymore if the slot doesn't match the ``plugin`` or ``database`` configuration options. Previously, the check for whether the slot matches those configuration options was not performed until after the replica copied the slot and started with it, resulting in unnecessary and repeated restarts.
```